aries_cloudagent/protocols/connections/v1_0/manager.py

```diff
@@ -190,6 +193,20 @@ async def create_invitation(
         )
         await connection.attach_invitation(self.context, invitation)
 
+        # MULTITENANCY: Add key to handled keys to associate new invitation to wallet.
+        ext_plugins = self.context.settings.get_value("external_plugins")
+        if ext_plugins and 'aries_cloudagent.wallet_handler' in ext_plugins:
+            wallet_handler: WalletHandler = await self.context.inject(WalletHandler)
+            wallet_id = self.context.settings.get_value("wallet.id")
+            await wallet_handler.add_connection(connection.connection_id, wallet_id)
+            await wallet_handler.add_key(connection.invitation_key, wallet_id)
+            post_filter = {"wallet_name": self.context.settings.get("wallet.name")}
+            wallet_records = await WalletRecord.query(
+                self.context)
+            wallet_record = wallet_recordes[0]
+            wallet_record.add_key(connection.invitation_key)
+            wallet_record.save(self.context)
+
```

Comment:
The last 4 lines would replace the ones above. Problem is that here we are in the "specific wallet context"(!=base) and therefore we  don't find  the records I think. Wallet_records are []